utils_nlp/models/transformers/common.py

```diff
@@ -34,9 +34,9 @@
 logger = logging.getLogger(__name__)
 
 
-def get_device(device, num_gpus, local_rank):
+def get_device(num_gpus=None, local_rank=-1):
```

Comment:
after this change, in `fine_tune` and `predict` do we still need the parameter device?

Reply:
> after this change, in fine_tune and predict do we still need the parameter device?

I think the device is computed in the child classes and then is passed into fine_tune and predict. We can leave it like this or maybe by DRY compute it in the base class, but it is not a big deal

Reply:
I'll leave it like this. It's not exposed to the user anyway.